From f007c14931084f216b93014bbe44ea018798f4d7 Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Sun, 26 Aug 2018 04:04:08 +0200
Subject: [PATCH 12/13] libbt-vendor: Fix BLUETOOTH_MAC_ADDR_BOOT_PROPERTY
 definition

HIDL interface still uses ro.boot.btmacaddr, so stick to that

Change-Id: Ica31816844ba147a22ea20c0ed2f0a4b0fad1592
---
 libbt-vendor/src/bt_vendor_qcom.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libbt-vendor/src/bt_vendor_qcom.c b/libbt-vendor/src/bt_vendor_qcom.c
index 35df9b0..8a16d2e 100644
--- a/libbt-vendor/src/bt_vendor_qcom.c
+++ b/libbt-vendor/src/bt_vendor_qcom.c
@@ -24,7 +24,7 @@
  *
  ******************************************************************************/
 #define LOG_TAG "bt_vendor"
-#define BLUETOOTH_MAC_ADDR_BOOT_PROPERTY "ro.vendor.boot.btmacaddr"
+#define BLUETOOTH_MAC_ADDR_BOOT_PROPERTY "ro.boot.btmacaddr"
 
 #include "bt_vendor_lib.h"
 #include "bt_vendor_persist.h"
-- 
2.17.1

